workbench/shared.php

```diff
@@ -273,6 +273,12 @@ function handleAllErrors($errno, $errstr, $errfile, $errline, $errcontext) {
     $errorId = basename($errfile, ".php") . "-$errline-" . time();
     workbenchLog(LOG_CRIT, "F",  "measure.fatal=1 " . $errorId . ":$errstr:" . print_r(debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS), true));
 
+    $request_errors = $_REQUEST['_request_real_errors'];
+    if (!isset($request_errors)) {
+        $request_errors = array();
+    }
+    array_push($request_errors, $errstr);
+    $_REQUEST['_request_real_errors'] = $request_errors;
```

Comment:
Don't know why it was not updated earlier... PHP is not my native language

Reply:
I mean I tried to init array and put it there but when I pushed vals there they weren't visible so I assume "by ref vs by value" happened